src/System.Diagnostics.Process/tests/ProcessTestBase.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Diagnostics.Tests
 {
     public class ProcessTestBase : RemoteExecutorTestBase
     {
-        protected const int WaitInMS = 600 * 1000;
+        protected const int WaitInMS = 30 * 1000;
```

Comment:
@tarekgh I think so too, I initially changed it to 10 but then thought about all of these slow CIs running these in parallel so increased to 30 just in case